src/shared/views/Mapper/index.js

```diff
@@ -432,9 +438,11 @@ export default class MapperScreen extends React.Component<Props> {
         // in which case, we use it as the tutorial, or fallback onto the default
         // tutorial content based on the project type
         let tutorialName;
-        if (projectObj.tutorialName !== undefined) {
-            tutorialName = projectObj.tutorialName;
+        if (projectObj.tutorialId !== undefined) {
+            tutorialName = projectObj.tutorialId;
         } else {
+            // TODO: thid default value should probably be removed entirely
```

Comment:
if you are using the new version of the app in production this will cause a crash since `build_area_tutorial` has categories instead of screen and so on.

To avoid this, you could just display a message like `A tutorial is not set up for this project at the moment` This is easier to understand for the users, but also from the code.